### PR TITLE
fix nightly releases & readme

### PR DESCRIPTION
**Pull request by Borda**
## What does this PR do?

switch making nightly from master
drop section about next 1.2 in readme

## Before submitting
- [ ] Was this discussed/approved via a GitHub issue? (not for typos and docs)
- [x] Did you read the [contributor guideline](https://github.com/PyTorchLightning/pytorch-lightning/blob/master/.github/CONTRIBUTING.md), **Pull Request** section?
- [x] Did you make sure your PR does only one thing, instead of bundling different changes together?
- [x] Did you make sure to update the documentation with your changes? (if necessary)
- [x] Did you write any new necessary tests? (not for typos and docs)
- [x] Did you verify new and existing tests pass locally with your changes?
- [x] Did you update the [CHANGELOG](https://github.com/PyTorchLightning/pytorch-lightning/blob/master/CHANGELOG.md)? (not for typos, docs, test updates, or internal minor changes/refactorings)

## PR review
Anyone in the community is free to review the PR once the tests have passed.
Before you start reviewing make sure you have read [Review guidelines](https://github.com/PyTorchLightning/pytorch-lightning/wiki/Review-guidelines). In short, see the following bullet-list:

 - [x] Is this pull request ready for review? (if not, please submit in draft mode)
 - [x] Check that all items from **Before submitting** are resolved
 - [x] Make sure the title is self-explanatory and the description concisely explains the PR
 - [x] Add labels and milestones (and optionally projects) to the PR so it can be classified
 - [x] **Check that target branch and milestone match!**

## Did you have fun?
Make sure you had fun coding 🙃

### CODE REVIEW

File: .github/workflows/ci_dockers.yml [hunk 1]
```diff
@@ -77,19 +77,20 @@ jobs:
...
             pytorch_version: 1.4
+          - python_version: 3.8
+            pytorch_version: 1.7
+          - python_version: 3.9
+            pytorch_version: 1.8
     steps:
       - name: Checkout
         uses: actions/checkout@v2
 
       # for PT 1.3 and 1.4 we need to use CUDA 10.1
```
Comment (carmocca) [inline .github/workflows/ci_dockers.yml:90]:
```suggestion
      # for PT 1.4 we need to use CUDA 10.1
```

File: .github/workflows/events-nightly.yml [hunk 2]
```diff
@@ -14,8 +14,8 @@ jobs:
     steps:
       # does nightly releases from feature branch
       - uses: actions/checkout@v2
-        with:
-          ref: release/1.2-dev
+        #with:
+        #  ref: release/1.2-dev
```
Comment (carmocca) [inline .github/workflows/events-nightly.yml:18]: might as well remove it

```suggestion
```

**Review by carmocca — APPROVED** [review]

File: README.md [hunk 3]
```diff
@@ -133,7 +136,7 @@ or nightly from testing PyPI
 pip install -iU https://test.pypi.org/simple/ pytorch-lightning
 ```
 
-<!-- end skipping PyPI description -->
+-->
```
Comment (carmocca) [inline README.md:139]: Why not remove it instead?

Comment (Borda) [reply README.md:139]: not sure how @edenlightning wants to formulate diff between bug-fix releases and true nightly

**Review by tchaton — APPROVED** [review]

**Review by kaushikb11 — APPROVED** [review]

File: README.md [hunk 4]
```diff
@@ -136,20 +137,33 @@ pip install pytorch-lightning
...
+  ![CI complete testing](https://github.com/PyTorchLightning/pytorch-lightning/workflows/CI%20complete%20testing/badge.svg?branch=release%2F1.1.x&event=push)
+  ![PyTorch & Conda](https://github.com/PyTorchLightning/pytorch-lightning/workflows/PyTorch%20&%20Conda/badge.svg?branch=release%2F1.1.x&event=push)
+  ![TPU tests](https://github.com/PyTorchLightning/pytorch-lightning/workflows/TPU%20tests/badge.svg?branch=release%2F1.1.x&event=push)
+  ![Docs check](https://github.com/PyTorchLightning/pytorch-lightning/workflows/Docs%20check/badge.svg?branch=release%2F1.1.x&event=push)
+
+  Install future release from the source
+  ```bash
+  pip install git+https://github.com/PytorchLightning/pytorch-lightning.git@release/1.1.x --upgrade
+  ```
+
```
[the remark this answers is not in the document]
Comment (Borda) [reply README.md:154]: yes, but still you want to see the status in the readme, and note that won't be in pypi decribtion

File: README.md [hunk 5]
```diff
@@ -136,20 +137,33 @@ pip install pytorch-lightning
...
   #### Install bleeding-edge - future 1.2
 
   Install future release from the source (no guarantees)
   ```bash
-  pip install git+https://github.com/PytorchLightning/pytorch-lightning.git@release/1.2-dev --upgrade
+  pip install git+https://github.com/PytorchLightning/pytorch-lightning.git@master
```
[the remark this answers is not in the document]
Comment (Borda) [reply README.md:159]: that is not the same, pypi won't be updated after each commit

[the remark this answers is not in the document]
Comment (carmocca) [reply README.md:159]: but `git+https://github.com/PytorchLightning/pytorch-lightning` pulls from github, not pypi

File: README.md [hunk 6]
```diff
@@ -136,20 +137,33 @@ pip install pytorch-lightning
   conda install pytorch-lightning -c conda-forge
   ```
 
+  #### Install stable/nightly - future 1.1.x
```
Comment (carmocca) [inline README.md:140]: nightly is the one right below

```suggestion
  #### Install stable - future 1.1.x
```

File: README.md (hunk 5 above)
[the remark this answers is not in the document]
Comment (Borda) [reply README.md:159]: right :]

**Review by edenlightning — APPROVED** [review]

**Pull request merged**